Source/Scene/Batched3DModel3DTileContentProvider.js

```diff
@@ -122,6 +122,9 @@ define([
         this.state = Cesium3DTileContentState.LOADING;
 
         loadArrayBuffer(this._url).then(function(arrayBuffer) {
+            if (that.isDestroyed()) {
+                throw new Error('tileset is destroyed');
```

Comment:
Yeah it can be silent. I did it this way so that the request scheduler and stats would get updated correctly, but since that all is changing anyways I think it's fine not to throw the errors.